scripts/uosc.lua

```diff
@@ -369,9 +370,13 @@ function update_human_times()
 		state.time_human = format_time(state.time)
 		if state.duration then
 			local speed = state.speed or 1
-			state.duration_or_remaining_time_human = format_time(
-				options.total_time and state.duration or ((state.time - state.duration) / speed)
-			)
+      if options.total_time or options.destination_time == "total" then
+        state.duration_or_remaining_time_human = format_time(state.duration)
```

Comment:
Rename `state.duration_or_remaining_time_human` everywhere in the codebase to `state.destination_time_human`.